### PR TITLE
Added JsCreatePromise to JsRT

### PR DESCRIPTION
**Pull request by kfarnung**
Retargeted the change to release/2.0

### CODE REVIEW

**Comment by kfarnung** [comment]
I figured out how to rebase #2581, closing this one.

File: lib/Jsrt/Jsrt.cpp [hunk 1]
```diff
@@ -4523,4 +4524,30 @@ CHAKRA_API JsRunSerialized(
...
+
+        PARAM_NOT_NULL(promise);
+        PARAM_NOT_NULL(resolve);
+        PARAM_NOT_NULL(reject);
+
+        *promise = nullptr;
+        *resolve = nullptr;
+        *reject = nullptr;
+
+        Js::JavascriptPromiseResolveOrRejectFunction *jsResolve;
```
Comment (akroshg) [inline lib/Jsrt/Jsrt.cpp]: nit: initialize this values with nullptr.

Comment (kfarnung) [inline lib/Jsrt/Jsrt.cpp]: I just pushed it, but I'll fix it.  I assume I should do the second one as well?

Comment (kfarnung) [inline lib/Jsrt/Jsrt.cpp]: PR #2595

**Pull request merged**